app/controllers/orders/orders_controller.rb

```diff
@@ -0,0 +1,68 @@
+class Orders::OrdersController < ApplicationController
+  include CurrentCart
```

Comment:
@b-artem you already have this module included in `ApplicationController`, you don't need to include it in class which was inherited from `ApplicationController`